examples/esp8266/esp8266.ino

```diff
@@ -66,6 +66,11 @@ void connect() {
     Serial.print(".");
   }
 
+  // do not verify tls certificate
```

Comment:
I hate to say this. But we may need to make this comment a little more detailed. Or, link somewhere that explains it. `setInsecure` could be scary for some. 

(I'm aware of what this setting does; I'm speaking from user's perspective)

Reply:
There is no documentation for `setInsecure` but what is in this comment is literally what it does. I'll add the comment from the link below this.